go.mod

```diff
@@ -1,20 +1,63 @@
 module github.com/rancher/eks-operator
 
-go 1.16
+go 1.19
```

Comment:
go 1.19 EOL with 1.21 being out, let's bump it to 1.20 if possible

Reply:
done, bumped to 1.20